turbinia/workers/photorec.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you also add a quick test for this?  It should be fairly lightweight, and you can look at the `testBulkExtractorRun()` method for an example since there is a base class you can extend for testing Tasks.

Reply:
Done.